### PR TITLE
Add ember-a11y-testing with ability to toggle a11yAudit acceptance tests

### PR DESCRIPTION
**Pull request by icole**
This adds support for ember-a11y-testing with acceptance test coverage of all of the routes that was requested in #233. 

By default each acceptance test per route will not run to avoid having failing tests in CI. Once a particular route has been addressed and all a11yAudit checks pass then adding `a11y: true` to the pages hash will run it as a normal test that expects a passing audit

I had tried to get this to work using QUnit `todo` as the default state, but unfortunately it appears testem does not support `todo` https://github.com/testem/testem/pull/1101

### CODE REVIEW

File: tests/acceptance/visual-routes-test.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      await prev;
+
+      await visit(config.route);
+
+      await percySnapshot(config.title);
+    }, Promise.resolve());
+  });
+
+  pages.forEach(function(page) {
+    (page.a11y_audit ? test : todo)(`testing a11y audit for ${page.route}`, async function (assert) {
```
Comment (dbollinger) [inline tests/acceptance/visual-routes-test.js]: 👍

File: tests/acceptance/visual-routes-test.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+import a11yAudit from 'ember-a11y-testing/test-support/audit';
+
+module('Acceptance | visual regression', function (hooks) {
+  setupApplicationTest(hooks);
+
+  // This is used to map the current **new** routes to what they used to be in the
+  // old ruby app. This is to allow us to compare what the new app looks like
+  // vs the old app.
+  const pages = [
+    { title: '/builds/', route: '/releases' },
```
Comment (jessica-jordan) [inline tests/acceptance/visual-routes-test.js]: Would it make sense to add a code comment here explaining the `a11y` switch? E.g.
```suggestion
    { title: '/builds/', route: '/releases'/*, a11y: true */ },  /* set the a11y property of a page object to true to run the audit for this particular route */
```

Comment (icole) [reply tests/acceptance/visual-routes-test.js]: I like this suggestion to add clarity. thanks!

**Comment by icole** [comment]
Thanks @jessica-jordan! Applied your suggestion and rebased from upstream. Let me know if I can do anything else.

**Pull request merged**